### ISSUE TITLE
New rule: meta viewport does not prevent zoom

### ISSUE THREAD

**Issue by carlosapaduarte** [w3c/wcag-act#455]
The ACT-Rules community group has completed work on "meta viewport does not prevent zoom". We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: meta viewport does not prevent zoom
**Rule id**: b4f0c3
**Link to the rule**: https://act-rules.github.io/rules/b4f0c3
**License**: W3C Software And Document Notice And License
**Implementors**:
- axe-core (Deque Systems)
- SortSite (PowerMapper)

**Comment by maryjom** [comment]
Here are the [survey results for "meta viewport does not prevent zoom"](https://www.w3.org/2002/09/wbs/93339/ACTViewZoom/results) completed on 4 June.

@WilcoFiers There seem to be several comments in this survey, but it was sent for a CFC to publish on 14 July. I don't see any issues opened for the comments, and unfortunately the day we discussed this was the one day we were unable to generate the meeting minutes for this meeting. Are no changes needed, or was there agreement to handle the issues after publication? I'm just trying to figure out what to put into the tracking table.

**Comment by WilcoFiers** [comment]
Rule has been approved, pull request for it is here: https://github.com/w3c/wcag-act-rules/pull/31
